boards/arm/mimxrt1050_evk/Kconfig.defconfig

```diff
@@ -15,6 +15,9 @@ endchoice
 config DISK_DRIVER_SDMMC
 	default y if DISK_DRIVERS
 
+config FLASH_MCUX_FLEXSPI_XIP
+	default y if FLASH
```

Comment:
```suggestion
	default y if FLASH
	depends on MEMC_MCUX_FLEXSPI
```

Reply:
Please try adding the below to this file and delete the selection from the defconfig
```
choice FLEXSPI_FLASH_DRIVER
       default FLASH_MCUX_FLEXSPI_HYPERFLASH if FLASH
endchoice
```